docs/source/get_involved/code_of_conduct.rst

```diff
@@ -168,15 +229,14 @@ GitHub.
 3.1 Contact points
 ~~~~~~~~~~~~~~~~~~
 
-If you feel able to, please contact Dr Franz Király by email at
-f.kiraly@ucl.ac.uk.
+Please report CoC incidents via the [reporting tool in the issue tracker](https://github.com/alan-turing-institute/sktime/issues/new/choose).
```

Comment:
```suggestion
Please report CoC incidents to info@sktime.org.
```

Reply:
I would actually introduce more "separation of powers", one flaw (in my opinion) of the current construction is the overlap between CC and CoC committee, due to potential conflicts of interest at leadership level.

In an ideal world, I think they should be staffed by distinct individuals, and have no overlap whatsoever.